libs/sdk/e2e-schematics/src/lib/send-cypress-screenshots-to-percy/send-cypress-screenshots-to-percy.spec.ts

```diff
@@ -0,0 +1,177 @@
+import { stripIndent } from '@angular-devkit/core/src/utils/literals';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import utils from '@percy/sdk-utils';
```

Comment:
There are no types available for this library.

Reply:
Paul started a convention to support packages without types. See the `/vendor/types/*` directory.